core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -47,6 +47,9 @@ def prepare_options
       @load_seed_data = options[:seed]
       @load_sample_data = options[:sample]
 
+      # Silence migrations output
```

Comment:
Let's reword this comment to avoid being specific about migrations. Setting the VERBOSE ENV to false might silence other things.

Reply:
I think you missed this comment @elia